app/repositories/elasticsearch_repository.py

```
@@ -0,0 +1,276 @@
+
+import logging
+from typing import List, Dict, Any
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import (
+    ConnectionError as ESConnectionError,
+    NotFoundError,
+    RequestError,
+)
+
+from app.core.settings import settings
+from app.utils.mongodb import transform_id_field
```

Flake8 (7.3.0)

[error] 'app.utils.mongodb.transform_id_field' imported but unused

(F401)